docker/ubuntu/Dockerfile

```diff
@@ -92,7 +94,10 @@ RUN  if [ -z "$WITH_TESTS" ] || [ "$WITH_TESTS" = "false" ]; then \
        -o cpack_name=dist \
        -o cpack_generator=TGZ \
        -pr "$CONAN_PROFILE" \
-       ${WITH_TESTS_ARG}
+       ${WITH_TESTS_ARG} \
```

Comment:
Nitpick.

```suggestion
       "$WITH_TESTS_ARG" \
```

Reply:
You have marked this as resolved, however, it seems not to be changed in the PR. Did you commit all your changes? :thinking: